ipam/allocator.go

```diff
@@ -623,7 +641,12 @@ func (alloc *Allocator) establishRing() {
 
 func (alloc *Allocator) createRing(peers []mesh.PeerName) {
 	alloc.debugln("Paxos consensus:", peers)
-	alloc.ring.ClaimForPeers(normalizeConsensus(peers))
+	alloc.ring.ClaimForPeers(normalizeConsensus(peers), alloc.isCIDRAligned)
+	// We assume that the peer has not possessed any address ranges before
+	err := alloc.monitor.HandleUpdate(nil, alloc.ring.OwnedAndMergedRanges())
```

Comment:
Suggest moving the various `HandleUpdate` calls inside `ringUpdated`.
Though the one in `Shutdown()` needs different handling - we don't want to retry pending ops there.